### DISCUSSION TITLE
How to run it after install

### DISCUSSION THREAD

**Discussion by malaymetaliya** [mishakorzik/AllHackingTools#78] [Q&A]
How to run it after install

**Comment by paodebatalha** [comment]
Idk too

**Comment by Rgb8746** [comment]
It's :
msdconsole

**Comment by Kha77led** [comment]
Ok

**Reply by Shaniji82** [reply]
Sent me

**Comment by Johnrey199427** [comment]
How to enter a licence to hack Facebook after installing

**Comment by ProjectOveriv3** [comment]
Do you people even read instructions or what ever? Or do you even know how for example "Termux" works? This is basic and i mean really basic understanding off Termux etc..
